### DISCUSSION TITLE
Updating replica count

### DISCUSSION THREAD

**Discussion by Ezinho031** [longhorn/longhorn#6242] [Q&A]
Hi!

I'm having trouble with a replica of a pvc from my Seq pod. The replica is in a constant loop of "rebuilding" and it's killing the kubernetes node where it is located.

What would happen if I reduce the replica count of the pvc from 3 to 2? Will it delete the replica in rebuilding state or just a random one from replica set?

**Comment by derekbit** [comment]
> it's killing the kubernetes node

Sorry, can you elaborate more on the statement?

IIRC, longhorn won’t automatically delete replicas after decreasing the
replica count?

Can you check longhorn-manager pods log to see why the replica is in a
rebuilding loop?
